bot/src/bounded-contexts/analytics/domain/services/processors/DistributionDataProcessor.ts

```
@@ -0,0 +1,305 @@
+  ): ChartData {
+    
+    const systemMap = new Map<string, { count: number; value: number }>();
+    
+    rawData.forEach(dataPoint => {
+      const systemName = dataPoint.systemName || dataPoint.solarSystemName || 'Unknown';
+      
+      if (!systemMap.has(systemName)) {
+        systemMap.set(systemName, { count: 0, value: 0 });
+      }
+      
+      const stats = systemMap.get(systemName)!;
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)